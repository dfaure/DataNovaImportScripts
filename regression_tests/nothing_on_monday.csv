#Identifiant;Libellé_du_site;Date_calendrier;Plage_horaire_1;Plage_horaire_2;Plage_horaire_3;Plage_horaire_4;Plage_horaire_5;Plage_horaire_6;Plage_horaire_7;Heure_limite_dépôt_Courrier;Heure_limite_dépôt_Chrono;Heure_limite_dépôt_Colis;Libellé_de_la_fermeture;Bp_Plage_horaire_1;Bp_Plage_horaire_2;Bp_Plage_horaire_3;Bp_Plage_horaire_4;Bp_Plage_horaire_5;Bp_Plage_horaire_6;Bp_Plage_horaire_7;Bp_Libellé_de_la_fermeture
14118D;NANTES CATHEDRALE RP;2022-03-01;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-15;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-04;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-03;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-02;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-08;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-16;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-29;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-03;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-01;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-30;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-02;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-31;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-05;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-10;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-07;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-08;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-06;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-12;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-13;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-10;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-09;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-26;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-09;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-11;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-22;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-25;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-17;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-23;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-27;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-19;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-20;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-18;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-24;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-13;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-12;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-05;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-06;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-14;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-29;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-15;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-23;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-16;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-20;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-17;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-19;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-06;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-07;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-12;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-27;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-21;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-24;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-22;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-28;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-03;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-17;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-18;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-01;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-26;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-04;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-10;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-28;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-14;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-11;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-08;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-20;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-21;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-05;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-30;09:30-12:30;;;;;;;10:00;10:00;10:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-27;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-15;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-13;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-29;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-24;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-25;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-19;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-26;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-02-28;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-16;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-23;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-22;FERME;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-31;08:00-12:30;14:30-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-06-01;08:00-12:30;14:00-19:00;;;;;;16:00;16:00;16:00;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-07;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-18;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-09;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-28;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-14;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-04;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-11;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-04-25;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-02;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-03-21;;;;;;;;;;;;FERME;;;;;;;
14118D;NANTES CATHEDRALE RP;2022-05-30;;;;;;;;;;;;FERME;;;;;;;
