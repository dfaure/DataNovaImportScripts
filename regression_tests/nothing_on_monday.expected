14118D|NANTES CATHEDRALE RP|Tu 08:00-12:30,14:30-19:00; We-Fr 08:00-12:30,14:00-19:00; Sa 09:30-12:30; PH off; 2022 May 26 08:00-12:30,14:00-19:00
